cv_origami/utils/validation.py

```diff
@@ -0,0 +1,146 @@
+try:
+    from pip.req import parse_requirements
+except ImportError:
+    # For pip version >= 10
+    from pip._internal.req import parse_requirements
+
+import os
+import zipfile
+
+from .file import get_model_bundles_base_dir, extract_zip_to_dir, \
+    clean_directory
+from ..exceptions import InvalidDemoBundleException, OrigamiConfigException
+from ..constants import REQUIREMENTS_FILE, ENTRYPOINT_PYTHON_MODULE, \
+    DOCKERFILE_FILE, ORIGAMI_ENV_FILE
+
+
+def validate_requirements_file(file_path):
+    """
+    Validate a python requirments.txt
+
+    It uses pip parse_requirements function to check whether
+    the requirements file in context is valid or not for parsing.
+
+    Args:
+        file_path (str): Absolute path to requirements file.
+
+    Raises:
+        InvalidDemoBundleException: Requirements file is not valid.
+    """
+    try:
+        reqs = parse_requirements(file_path, session=False)
+        requirements = []
+        for r in reqs:
+            requirements.append(r.req)
+        if not requirements:
+            raise InvalidDemoBundleException("Requirements file is empty.")
+    except Exception as e:
+        raise InvalidDemoBundleException("Requirements file is not valid")
+
+
+def validate_dockerfile(file_path):
```

Comment:
Then how are you thinking to do this?

Reply:
For now, I don't have any way to do this. But it doesn't matter much as it will be checked during demo deployment while building the image. I will add an implementation to this function if I get something useful.